Dockerfile-velero-ppc64le

```diff
@@ -0,0 +1,32 @@
+# Copyright 2017, 2019 the Velero contributors.
```

Comment:
This can be just 2019 rather than 2017, 2019.

Reply:
fixed